huey/storage.py

```diff
@@ -104,9 +104,18 @@ def flush_all(self):
 
 
 class RedisStorage(BaseStorage):
-    def __init__(self, name='huey', blocking=False, read_timeout=1,
-                 max_errors=1000, connection_pool=None, **connection_params):
-        if connection_pool is None:
+    def __init__(self, name='huey', blocking=False, read_timeout=1, max_errors=1000,
+            connection_pool=None, url=None, **connection_params):
+        if sum(bool(x) for x in (url, connection_pool, connection_params)) > 1:
```

Comment:
You're relying on the quirk that Python considers `True` to be `1` -- that is why this trick works. I think it is quite obscure, though...summing booleans.

Since `filter()` isn't as nice because fucking stupid ass Python3, how about changing this to a list comprehension?

```python
if len(item for item in (url, connection_pool, connection_params)) > 1:
```

Would you consider changing that?